packages/core/src/time-field/time-field-hidden-input.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ComponentProps } from "solid-js";
+
+import { useFormControlContext } from "../form-control";
+import { useTimeFieldContext } from "./time-field-context";
+
+export interface TimeFieldHiddenInputProps extends ComponentProps<"input"> {}
+
+export function TimeFieldHiddenInput(props: TimeFieldHiddenInputProps) {
+	const formControlContext = useFormControlContext();
+	const context = useTimeFieldContext();
+
+	return (
+		<input
+			type="text"
+			hidden
```

Comment:
This should use `style={visuallyHiddenStyles}` instead, this is still recognized as visible by browsers but doesn't actually show. Should also be aria-hidden.